cypress/integration/brewery/ScenarioManager.spec.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import utils from '../../commons/TestUtils';
+import { DATASET, RUN_TEMPLATE, SCENARIO_STATUS } from '../../commons/constants/brewery/TestConstants';
+import { Scenarios, ScenarioManager, ScenarioParameters, Login } from '../../commons/actions';
+
+describe('Create scenario and check its data in scenario manager', () => {
+  Cypress.Keyboard.defaults({
+    keystrokeDelay: 0,
+  });
+
+  const runTemplate = RUN_TEMPLATE.BASIC_TYPES;
+
+  function forgeScenarioName() {
+    const prefix = 'Test Cypress scenario manager - ';
+    const randomString = utils.randomStr(7);
+    return prefix + randomString;
+  }
+
+  const scenarioNamesToDelete = [];
+
+  before(() => {
+    Login.login();
+  });
+
+  beforeEach(() => {
+    Login.relogin();
+  });
+
+  after(() => {
+    ScenarioManager.deleteScenarioList(scenarioNamesToDelete);
+  });
+
+  it('Check scenario in scenario manager', () => {
+    const scenarioName = forgeScenarioName();
+    scenarioNamesToDelete.push(scenarioName);
+
+    Scenarios.createScenario(scenarioName, true, DATASET.BREWERY_ADT, runTemplate).then((req) => {
+      const scenarioId = req.scenarioCreatedId;
+      const scenarioOwnerName = req.scenarioCreatedOwnerName;
+      const scenarioCreationDate = req.scenarioCreatedCreationDate;
+
+      ScenarioManager.switchToScenarioManager();
+
+      ScenarioManager.getScenarioAccordion(scenarioId).click();
+      ScenarioManager.getScenarioOwnerName(scenarioId).should('have.text', scenarioOwnerName);
+      ScenarioManager.getScenarioCreationDate(scenarioId).should('have.text', scenarioCreationDate);
+      ScenarioManager.getScenarioEditableLabel(scenarioId).should('have.text', scenarioName);
+      ScenarioManager.getScenarioRunStatus(scenarioId, SCENARIO_STATUS.CREATED);
+      ScenarioManager.getScenarioDataset(scenarioId).should('have.text', DATASET.BREWERY_ADT, { matchCase: false });
+
+      Scenarios.switchToScenarioView();
+
+      ScenarioParameters.updateAndLaunch(false, true);
+
+      ScenarioManager.switchToScenarioManager();
+
+      ScenarioManager.getScenarioAccordion(scenarioId).click();
+      ScenarioManager.getScenarioRunStatus(scenarioId, SCENARIO_STATUS.RUNNING);
+      ScenarioManager.getScenarioRunStatus(scenarioId, SCENARIO_STATUS.SUCCESSFUL, 60);
```

Comment:
I'm afraid 60 seconds might not be enough when the API is overwhelmed. We're using 300 seconds in the test _FileParameters-tests.spec.js_.